### ISSUE TITLE
Subscription criteria wrong for multi-value criteria

### ISSUE THREAD

**Issue by ahatherly** [nhsconnect/Events-Management#48]
I think the syntax currently used in the subscription criteria for NEMS may be incorrect. The criteria attribute is supposed to use the same syntax as standard FHIR searches. In a FHIR search if you want to search on a field and specify that the field should be one of two values, this is done using a comma separated list after the =. So, for example to search for events with either event type A or B you would use this syntax:
MessageHeader.event=A,B
However, the current spec says you should specify the criteria twice like this:
MessageHeader.event=A&MessageHeader.event=B
This would actually be treated as an AND in FHIR, meaning it would only match events with a code of both A and B (which is impossible)..
I realise this may actually have originally been my error, but I thought I should bring it to your attention in any case...

**Comment by james-answer** [comment]
I agree this should be done as a comma separated list and currently does not logically make sense. I will add it to our list of things to look at changing.
